### PR TITLE
get rid of warnings

### PR DESCRIPTION
**Pull request by VitalyyP**
- fix show warnings when running eslint;
- set ariaLabelContest prop in IconWithClassName as unrequited;

### CODE REVIEW

File: src/components/common/customImage/CustomImage.tsx [hunk 1]
```diff
@@ -5,9 +5,16 @@ import getImageQuality from 'src/helpers/getImageQuality';
 interface CustomImageProps extends Omit<ImageProps, 'quality'> {}
 
 const CustomImage = (props: CustomImageProps): JSX.Element => {
-  const { src, ...rest } = props;
+  const { src, alt, ...rest } = props;
```
[the remark this answers is not in the document]
Comment (VitalyyP) [reply src/components/common/customImage/CustomImage.tsx:8]: Done

**Review by yuramax — COMMENTED** [review]
@VitalyyP, “npm run lint” is now working without any problems! Great job! 👍 

I have a question about “npm install”, it shows another warning related to eslint, do you think it should be included in this task or create a separate task? Thanks.

npm-install

**Review by yuramax — APPROVED** [review]
Looks great! 👍 Thank you.

**Review by Anton-88 — APPROVED** [review]
Well done!

**Pull request merged**